recipe/build.sh

```diff
@@ -39,6 +39,7 @@ go-licenses csv ${module}
 EOF
     go-licenses csv "${module}" | sort >> "${output}"
     go-licenses save "${module}" --save_path="${tmp_dir}"
+    chmod -R +rw "${acc_dir}"
```

Comment:
Without it I got
```
cp: cannot create regular file `/home/conda/feedstock_root/build_artifacts/cni-plugins_1604485827948/work/gather-licenses-acc/github.com/containernetworking/cni/pkg/LICENSE': Permission denied
```
but I have no clue why.
Let's see if this `chmod` changes anything..

Reply:
That's really strange. But `chmod` made it work.
I don't get this with a local (also container-based) build. Maybe `umask` or something is different on CI? I'm confused.